packages/client/src/components/Actions/components/Confirmations.js

```diff
@@ -0,0 +1,46 @@
+import Svg from "library/Svg";
+import { useContacts } from "hooks";
+import { getSafeContactName, shortenAddr, shortenName } from "utils";
+
+const Confirmations = ({ confirmations, safeData }) => {
+  const { address: safeAddress } = safeData;
+
+  const { contacts } = useContacts(safeAddress);
+
+  const getNumberOfConfirmations = () => {
+    const numberApproved = Object.keys(confirmations).filter(
+      (key) => confirmations[key] === "approved"
+    )?.length;
```

Comment:
Probably don't need the optional chain here, if confirmations is an empty object, Object.keys(confirmations) will return an empty array, filtering it returns an empty array as well
If we want to check against nullish value of confirmations, we can do
`Object.keys(confirmations ?? [])`